### ISSUE TITLE
!inter not working

### ISSUE THREAD

**Issue by ketarXVII** [ScalaWilliam/ActionFPS#398]
Title is kinda self-explanatory.

When I call for !inter on aura servers (of course with my registered nickname) I do not receive any notification from https://actionfps.com/

at the same time I see that my calls are not echoed by inter-bot on discord afps/#inters channel. 
Other ppl told me they are experiencing the same issue

**Comment by ScalaWilliam** [comment]
Here's the error:

```
java.util.concurrent.ExecutionException: Boxed Error
Jul 06 18:34:16 odin.duel.gg run.sh[28829]: at scala.concurrent.impl.Promise$.resolver(Promise.scala:83)
Jul 06 18:34:16 odin.duel.gg run.sh[28829]: at scala.concurrent.impl.Promise$.scala$concurrent$impl$Promise$$resolveTry(Promise.scala:75)
Jul 06 18:34:16 odin.duel.gg run.sh[28829]: at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:280)
Jul 06 18:34:16 odin.duel.gg run.sh[28829]: at scala.concurrent.Promise.complete(Promise.scala:49)
Jul 06 18:34:16 odin.duel.gg run.sh[28829]: at scala.concurrent.Promise.complete$(Promise.scala:48)
Jul 06 18:34:16 odin.duel.gg run.sh[28829]: at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:183)
Jul 06 18:34:16 odin.duel.gg run.sh[28829]: at af.inters.DiscordInters$stateMachine$macro$1$1.apply(DiscordInters.scala:41)
Jul 06 18:34:16 odin.duel.gg run.sh[28829]: at af.inters.DiscordInters$stateMachine$macro$1$1.apply$mcV$sp(DiscordInters.scala:41)
Jul 06 18:34:16 odin.duel.gg run.sh[28829]: at af.inters.DiscordInters$stateMachine$macro$1$1.apply(DiscordInters.scala:41)
Jul 06 18:34:16 odin.duel.gg run.sh[28829]: at af.inters.DiscordInters$stateMachine$macro$1$1.apply(DiscordInters.scala:41)
Jul 06 18:34:16 odin.duel.gg run.sh[28829]: Caused by: java.lang.NoSuchMethodError: play.api.libs.ws.package$.writeableOf_JsValue()Lplay/api/libs/ws/BodyWritable;
Jul 06 18:34:16 odin.duel.gg run.sh[28829]: at af.inters.DiscordInters$stateMachine$macro$1$1.apply(DiscordInters.scala:61)
Jul 06 18:34:16 odin.duel.gg run.sh[28829]: at af.inters.DiscordInters$stateMachine$macro$1$1.apply$mcV$sp(DiscordInters.scala:41)
Jul 06 18:34:16 odin.duel.gg run.sh[28829]: at af.inters.DiscordInters$stateMachine$macro$1$1.apply(DiscordInters.scala:41)
Jul 06 18:34:16 odin.duel.gg run.sh[28829]: at af.inters.DiscordInters$stateMachine$macro$1$1.apply(DiscordInters.scala:41)
Jul 06 18:34:16 odin.duel.gg run.sh[28829]: at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:653)
Jul 06 18:34:16 odin.duel.gg run.sh[28829]: at scala.util.Success.$anonfun$map$1(Try.scala:251)
Jul 06 18:34:16 odin.duel.gg run.sh[28829]: at scala.util.Success.map(Try.scala:209)
Jul 06 18:34:16 odin.duel.gg run.sh[28829]: at scala.concurrent.Future.$anonfun$map$1(Future.scala:287)
Jul 06 18:34:16 odin.duel.gg run.sh[28829]: at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:29)
Jul 06 18:34:16 odin.duel.gg run.sh[28829]: at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:29)
```

**Comment by ScalaWilliam** [comment]
Fixed in 63a052fcb41e23813c678bfc2ac1237430009756
